doc/access-logs.md

```diff
@@ -1,5 +1,43 @@
 ## Understanding Access Logs
 
+### Fields
+
+* `app_id`: Cloud Foundry application GUID
+* `authority`: Authority (for HTTP/2) or Host header for HTTP/1.1 of the request; see [RFC7540](https://tools.ietf.org/html/rfc7540#section-8.1.2.3) for more details
+* `bytes_received`: Body bytes received, [%BYTES_RECEIVED% in Envoy](https://www.envoyproxy.io/docs/envoy/latest/configuration/observability/access_log.html?highlight=BYTES_RECEIVED)
+* `bytes_sent`: Body bytes sent. For WebSocket connection it will also include response header bytes; [%BYTES_SENT% in Envoy](https://www.envoyproxy.io/docs/envoy/latest/configuration/observability/access_log.html?highlight=BYTES_SENT)
+* `downstream_local_address`: Local address of the downstream connection; see [%DOWNSTREAM_LOCAL_ADDRESS% in Envoy docs](https://www.envoyproxy.io/docs/envoy/latest/configuration/observability/access_log.html?highlight=DOWNSTREAM_LOCAL_ADDRESS),
+* `downstream_remote_address`: Remote address of the downstream connection; see [%DOWNSTREAM_REMOTE_ADDRESS% in Envoy docs](https://www.envoyproxy.io/docs/envoy/latest/configuration/observability/access_log.html?highlight=DOWNSTREAM_REMOTE_ADDRESS),
+* `duration`: Total duration in milliseconds of the request from the start time to the last byte out; see [%DURATION% in Envoy docs](https://www.envoyproxy.io/docs/envoy/latest/configuration/observability/access_log.html?highlight=DURATION),
+* `method`: Method of HTTP request
+* `organization_id`:  Cloud Foundry organization GUID of the application
+* `path`: HTTP request path
+* `process_type`: `Procfile` process type
```

Comment:
I would phrase this as something like: 

[Cloud Foundry Process Type](https://v3-apidocs.cloudfoundry.org/version/3.80.0/index.html#processes) for the backend being routed to.

They can be set in other ways than a `Procfile`.